pandas/core/arrays/categorical.py

```diff
@@ -2494,6 +2494,20 @@ def isin(self, values):
         code_values = code_values[null_mask | (code_values >= 0)]
         return algorithms.isin(self.codes, code_values)
 
+    def replace(self, to_replace, value, inplace=False):
+        inplace = validate_bool_kwarg(inplace, "inplace")
```

Comment:
cool, can you type & add a doc-string?

Reply:
fixed (only added types for the boolean arguments in both replace functions, do the `to_replace` and `value` parameters need types?)